ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/Server.java

```diff
@@ -240,30 +243,32 @@ private static HttpServerOptions createHttpServerOptions(final ApiServerConfig a
     if (tls) {
       options.setUseAlpn(true).setSsl(true);
 
-      final String keyStorePath = apiServerConfig.getString(ApiServerConfig.TLS_KEY_STORE_PATH);
-      final String keyStorePassword = apiServerConfig
-          .getString(ApiServerConfig.TLS_KEY_STORE_PASSWORD);
+      final String keyStorePath = ksqlRestConfig
+          .getString(SslConfigs.SSL_KEYSTORE_LOCATION_CONFIG);
+      final Password keyStorePassword = ksqlRestConfig
+          .getPassword(SslConfigs.SSL_KEYSTORE_PASSWORD_CONFIG);
       if (keyStorePath != null && !keyStorePath.isEmpty()) {
         options.setKeyStoreOptions(
-            new JksOptions().setPath(keyStorePath).setPassword(keyStorePassword));
+            new JksOptions().setPath(keyStorePath).setPassword(keyStorePassword.value()));
```

Comment:
Do we need a null check on `keyStorePassword` here (and on `trustStorePassword` below)? The old code had a null check that's been removed. Haven't looked into whether it's actually needed or not.

Reply:
I don't think it can be null as the default is "".